demos/http/http_demo_mutual_auth/http_demo_mutual_auth.c

```diff
@@ -79,30 +85,20 @@
     #define USER_BUFFER_LENGTH    ( 2048 )
 #endif
 
-/**
- * @brief The length of the HTTP server host name.
- */
-#define AWS_IOT_ENDPOINT_LENGTH               ( sizeof( AWS_IOT_ENDPOINT ) - 1 )
-
-/**
- * @brief The length of the ALPN protocol name.
- */
-#define IOT_CORE_ALPN_PROTOCOL_NAME_LENGTH    ( sizeof( IOT_CORE_ALPN_PROTOCOL_NAME ) - 1 )
```

Comment:
The mqtt mutual auth demo has the alpn defined in the c file because it does not change. Could you move that here?